### PR TITLE
[14.0][ADD] website_sale_ajaxify_cart

### PR DESCRIPTION
**Pull request by geomer198**
This module extends functionality of website sale module by providing Ajax-based add-to-cart functionality. It allows customers to add products to their cart without leaving the page.

### CODE REVIEW

**Review by GabbasovDinar — APPROVED** [review]
Code LGTM

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Comment by francesco-ooops** [comment]
@OCA/e-commerce-maintainers good for merge?

**Comment by francesco-ooops** [comment]
@pedrobaeza good for you? :)

**Comment by pedrobaeza** [comment]
I can't say about the functionality, so an e-commerce PSC should check it.

**Comment by francesco-ooops** [comment]
@simahawk if you have 5 mins :)

**Comment by francesco-ooops** [comment]
@simahawk ping!

**Comment by francesco-ooops** [comment]
@OCA/e-commerce-maintainers any good soul for review? 🙏

**Comment by francesco-ooops** [comment]
@OCA/e-commerce-maintainers whenever possible :)

**Comment by francesco-ooops** [comment]
@yajo could you please merge this one?

File: website_sale_ajaxify_cart/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "eCommerce Ajaxify cart",
+    "summary": "eCommerce Ajaxify cartr",
```
Comment (yajo) [inline website_sale_ajaxify_cart/__manifest__.py:3]: As a human-readable name this is a bit cryptic. You might consider to improve it.

**Review by yajo — APPROVED** [review]
Code & functional review.

/ocabot merge patch

**Pull request merged**